backend/app/vending/__init__.py

```diff
@@ -226,7 +226,9 @@ def status(login=Depends(login_state)) -> VendingStatus:
 
 
 @router.post("/status/onboarding", tags=["vending"])
-def start_onboarding(data: VendingOnboardingRequest, login=Depends(login_state)):
+def start_onboarding(
```

Comment:
GitHub prevents me commenting on unchanged lines, but on L207 of this file, the return type should match what is declared in the function header. It's currently causing issues with vending link retrieval in the frontend (a query condition falsely becomes true and tries to proceed to dashboard link retrieval).

Reply:
I fixed in testing locally by just returning a default `VendingStatus` with `status="none"` or similar and all `False` values.

Was unable to push to your fork branch.

Reply:
I really don't like that status is just a string :/

Reply:
But changed it